ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -47,6 +51,7 @@ class ShopperInsightsClient @VisibleForTesting internal constructor(
                     )
                 )
             )
+            braintreeClient.sendAnalyticsEvent(GET_RECOMMENDED_PAYMENTS_FAILED)
```

Comment:
Would it be possible to put why it failed? Merchant passing bad data

Reply:
Max, I don't see us sending any data in any of the other places. We'll have to look into the API to figure out if it accepts a key, value data object. I'll keep this PR as is and figure that out as part of another ticket if the need arises.

Reply:
Makes sense.